docs/install/install-staging-migrate.md

```diff
@@ -490,6 +494,24 @@ This step will show you how to shell into your currently running Staging Apache
 | - Allow vpnkit.exe to communicate with the network.  Click Okay or Allow (accept default selection).|
 | - If the process seems to halt, check the taskbar for background windows.|
 
+**install_solution_packs.sh: installs Islandora solution packs**
+
+Since you've imported an existing Drupal database, you must now reinstall the Islandora solution packs so the Fedora repository will be ready to ingest objects.
+
+* Copy the "install_solution_packs.sh" to the root of the Drupal directory on your Apache container
+    * `docker cp scripts/apache/install_solution_packs.sh your-apache-containername:/var/www/html/install_solution_packs.sh`
+* Change the permissions on the script to make it executable
+    * **For Mac/Ubuntu/CentOS/etc:** `docker exec -it your-apache-containername bash -c "chmod +x /var/www/html/install_solution_packs.sh"`
+    * **For Microsoft Windows:** `winpty docker exec -it your-apache-containername bash -c "chmod +x /var/www/html/install_solution_packs.sh"`
+* Run the script
+    * **For Mac/Ubuntu/CentOS/etc:** `docker exec -it your-apache-containername bash -c "cd /var/www/html && ./install_solution_packs.sh"`
+    * **For Microsoft Windows:** `winpty docker exec -it your-apache-containername bash -c "cd /var/www/html && ./install_solution_packs.sh"`
```

Comment:
Is this the command if using Git Bash for Windows?  The instructions assume that's the terminal being used, and if I recall correctly, we don't need to preface commands with winpty.